lib/utils.pm

```diff
@@ -617,7 +617,8 @@ sub validate_repos {
                     $uri = "http[s]*://.*suse";
                 }
                 elsif (get_var('USBBOOT')) {
-                    $uri = "hd:///.*usbstick";
+                    $uri = "hd:///.*usb" if get_var('VERSION', '12-SP3');
+                    $uri = "hd:///.*usbstick" if !get_var('VERSION', '12-SP3');    #pre-SP3 check
```

Comment:
that's not a good idea. Why shouldn't it apply for more recent versions? Probably you want to use `utils::sle_version_at_least`